ubuntudesign/www.ubuntu.com
# Move maas

## Done

* moved maas.html from /cloud to /server
* renamed it in the menu from MAAS to Server provisioning
* updated all the redirects
* updated the contact us in the contextual footer to the server on
* fixed up the _server.scss to inherit better and moved maas page styling there and out of _cloud.scss

## QA

1. go to /server/maas and see that it looks like the live /cloud/maas
2. see that the menu name is 'Server provisioning'
3. see that the contextual footer is now the server contact-us

## Issue / Card

[trello card](https://trello.com/c/40W4AhLa)
